web/src/main/resources/static/app.js

```diff
@@ -49,25 +48,19 @@ const ROUTE_ICON_MAPPING = {
   '4': BoatMarkerIcon,
 }
 
-const randomColor = () => "#" + ((1<<24)*Math.random() | 0).toString(16)
-
 let COLOR_PALETTE = {
   PER_ROUTE: {},
   PER_AGENCY: {}
 }
 
+let KNOWN_ROUTES = {}
+
+const randomColor = () => "#" + ((1 << 24) * Math.random() | 0).toString(16)
+
 function currentTime() {
-  if (DEBUG && DEBUG_REPLAY) {
-    var dateOffset = ((149)*60*60*1000);
-    var fakeNow = new Date();
-    fakeNow.setTime(fakeNow.getTime() - dateOffset);
-    return fakeNow
-  }
   return new Date();
```

Comment:
I don't follow JS current patterns, but is the ending semicolon mandatory?

Reply:
it's a battle no-semicolon-team is currently winning. as an ex-pascal guy i'm not happy about that. will lint with more strict settings to be consistent...

Reply:
Yes, I'd rather have semi-colons everywhere but the most important is consistency